### ISSUE TITLE
bot orders removed after leaving outpost

### ISSUE THREAD

**Issue by LePiromano** [Regalis11/Barotrauma#8405]
- [yes] I have searched the issue tracker to check if the issue has already been reported.

**Description**
![image](https://user-images.githubusercontent.com/55637865/155898574-3e1ef032-56f7-4d6d-ac5e-f55b7600d426.png)

**Steps To Reproduce**
1- give orders in outpost
2- leave outpost
3- error and orders removed

**Version**
v0.17.0.0

**Comment by LePiromano** [comment]
actually they're removed when leaving any location, not just outposts

**Comment by DrinkFromTheCup** [comment]
+1
Same errors in console, same results.
Unstable, today's latest.

**Comment by ekqvist** [comment]
Thanks for the report! I hope to have addressed the issue with the following fixes:

- Fixed report buttons creating orders with the incorrect target type (Regalis11/Barotrauma-development@89a5fd3)
- Fixed command interface order nodes creating orders with the incorrect target type (Regalis11/Barotrauma-development@ea8f284)
- Fixed command interface shortcut nodes creating orders with the incorrect target type (Regalis11/Barotrauma-development@76e5083)
- Fixed command interface contextual order nodes creating orders with the incorrect target type (Regalis11/Barotrauma-development@a0fac63)
- Fixed the AI commander creating some orders with the incorrect target type (Regalis11/Barotrauma-development@ed1a909)

**Comment by mrjpjr** [comment]
Yes, I am also getting this issue.
![order error](https://user-images.githubusercontent.com/58775950/157439325-7c45f3c1-aa44-4d60-9128-e67ac0f3e29c.jpg)
It started happening after I changed the longstanding orders that I gave the bots.
(e.g. changed the mechanic's order from 'fix leaks' and 'repair mechanical systems' to 'fix leaks' and 'repair electrical systems')

My save is from version 16.3 and I (most likely assigned) the original orders since that version, so maybe the issue might be related to the older save?

Going to try removing all orders and issuing fresh ones in a new round
